src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import '../styles/App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
+
+function App() {
+  return (
+    <>
+      <Display result={calculate} />
```

Comment:
Please check this in order to fix the prop-type error. I think `result` is not supposed to be a function.